tools/lsp/ui/components/property-widgets.slint

```diff
@@ -302,14 +302,14 @@ component StringWidget {
     callback code-action();
     callback reset-action();
 
-    callback test-string-binding(text: string, is-translatable: bool, tr-context: string, tr-plural: string, tr-plural-expression: string) -> bool;
-    callback set-string-binding(text: string, is-translatable: bool, tr-context: string, tr-plural: string, tr-plural-expression: string);
+    callback test-string-binding(text: string, is-translated: bool) -> bool;
+    callback set-string-binding(text: string, is-translated: bool);
 
     function tsb() -> bool {
-        return test-string-binding(text-rle.text, tr-cb.checked, tr-context.text, tr-plural.text, tr-plural-expression.text);
+        return test-string-binding(Api.string-to-code(text-rle.text, tr-cb.checked, tr-context.text, tr-plural.text, tr-plural-expression.text), tr-cb.checked);
```

Comment:
Btw, the plural handling doesn't work. Maybe it'd be best to hide the "Plural" and "Expression" widgets.